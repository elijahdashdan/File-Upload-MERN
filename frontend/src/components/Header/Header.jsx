import React from "react";

function Header() {
  return (
    <div data-testid = 'header' className="header">
      <h1>fileUploader</h1>
    </div>
  );
}

export default Header;
