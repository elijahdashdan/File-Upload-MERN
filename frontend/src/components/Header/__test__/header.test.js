import React from 'react';
import ReactDOM from 'react-dom';
import Header from '../Header';
import {render} from '@testing-library/react';

it("Renders without Crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Header />,div)
    ReactDOM.unmountComponentAtNode(div)
})

it("Renders Headers component correctly", () => {
    render()
})