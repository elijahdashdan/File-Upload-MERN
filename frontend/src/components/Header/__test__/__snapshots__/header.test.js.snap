// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <h1>
    fileUploader
  </h1>
</div>
`;
